amazon/ion/simpleion.py

```diff
@@ -155,7 +156,56 @@ def dumps(obj, imports=None, binary=True, sequence_as_stream=False,
     return ret_val
 
 
-def load(fp, catalog=None, single_value=True, parse_eagerly=True, text_buffer_size_limit=None):
+class IonPyValueModel(IntEnum):
+    """Flags to control the types of values that are emitted from load(s).
+
+    The flags may be mixed so that users can intentionally demote certain Ion
+    constructs to standard Python types. In general this will increase read
+    performance and improve interoperability with other Python code at the
+    expense of data fidelity.
+
+    For example:
+        value_model = IonPyValueModel.MIXED | IonPyValueModel.SYMBOL_AS_TEXT
+
+    would mean that any Symbols without annotations would be emitted as str,
+    any Symbols with annotations would be emitted as IonPyText(IonType.SYMBOL).
+
+    todo: add/extend this as desired. some possible additions:
+        CLOB_AS_BYTES
+        SEXP_AS______ (list, tuple, either?)
+        IGNORE_ANNOTATIONS
+        TIMESTAMP_AS_DATETIME
+        ALWAYS_STD (union of all flags)
+    """
+
+    ION_PY = 0
+    """All values will be instances of the IonPy classes."""
+
+    MIXED = 1
+    """Values will be of the IonPy classes when needed to ensure data fidelity,
+    otherwise they will be standard Python types.
+    """
+
+    SYMBOL_AS_TEXT = 2
+    """Symbol values will be IonPyText(IonType.SYMBOL) or str if bare.
+    
+    Symbol Ids are always lost if present. When bare values are emitted, the
```

Comment:
An error will have to be raised if the symbol has unknown text (the only case where the symbol ID (really, the import location)) matters.

Reply:
FTR we're already losing strict fidelity for this case, which is I suppose my rebuttal to your comment on https://github.com/amazon-ion/ion-python/issues/316

Reply:
I see. That currently raises an error as well, though we could be returning a SymbolToken with None text and Sid and import location.